CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,56 @@
+# 📜 Code of Conduct  
+
+## 👥 Our Pledge  
+
+We, as contributors and maintainers of this project, pledge to create a **welcoming, inclusive, and harassment-free** environment for everyone, regardless of age, gender, race, ethnicity, disability, level of experience, or any other characteristic.  
+
+We expect all members of the **Google Summer of Code @ WSO2** community to follow this Code of Conduct in all interactions related to this project.  
+
+---
+
+## 🌟 Our Standards  
+
+To foster a **positive and inclusive** environment, we encourage:  
+
+✅ **Be respectful** – Treat others with kindness and empathy.  
+✅ **Be collaborative** – Help others and be open to feedback.  
+✅ **Be professional** – Use appropriate language and avoid offensive remarks.  
+✅ **Give constructive criticism** – Provide helpful feedback with kindness.  
+✅ **Respect differences** – Different perspectives and experiences make us stronger.  
+
+We do **not** tolerate:  
+
+🚫 **Harassment** – Any form of discrimination, abuse, or threats.  
+🚫 **Trolling & Personal Attacks** – Bullying, insults, or spreading negativity.  
+🚫 **Spam & Disruptive Behavior** – Unwanted promotion, excessive self-promotion, or repeated disruptions.  
+🚫 **Exclusionary Language** – Racist, sexist, homophobic, or otherwise discriminatory language.  
+
+---
+
+## ⚠️ Reporting Issues  
+
+If you experience or witness any behavior that violates this Code of Conduct:  
+
+1. **Contact the repository maintainers** at `[your contact email]`.  
+2. **Report the issue via GitHub Issues**, if appropriate.  
+3. **Respect confidentiality** – Reports will be handled **privately** and **fairly**.  
```

Comment:
As discussed let's add this as a separate item instead of including it in the list.

Reply:
Fixed with [7a8874d](https://github.com/wso2/google-summer-of-code/pull/1/commits/7a8874d16a002d917f69b3089965e4809f15be40)